rancid-tomatillos/src/containers/Login/Login.js

```diff
@@ -0,0 +1,86 @@
+import React, { Component }from 'react';
+import './Login.scss';
+import { connect } from 'react-redux';
+import { Redirect } from 'react-router-dom';
+import { addUser, updateLoggedIn } from '../../actions/index';
+
+
+class Login extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { 
+      email: '',
+      password: '',
+      error: false 
+     }
+  }
+
+
+  handleChange = (e) => {
+    this.setState({[e.target.name]: e.target.value})
+  }
+
+  handleSubmit = () => {
+    const options = {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json'
+      },
+      body: JSON.stringify(this.state)
+    }
+
+    fetch('https://rancid-tomatillos.herokuapp.com/api/v1/login', options)
+      .then(res => {
+        if(!res.ok) {
+         throw Error('Incorrect Username/Password')
+        }
+        return res.json()
+      })
+      .then(data => {
+        console.log(data)
+        this.props.addUser(data)
```

Comment:
We could probably deconstruct props at the beginning of this function for greater readability when we use those props